contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/ImportFromChannelsModal.vue

```diff
@@ -3,9 +3,7 @@
   <FullscreenModal v-model="dialog" :header="headerText">
     <template v-if="isReview" #close>
       <VBtn icon @click.stop="goBackToBrowse">
-        <Icon class="rtl-flip">
-          arrow_back
-        </Icon>
+        <Icon class="rtl-flip" icon="back" :color="$themeTokens.textInverted" />
```

Comment:
`rtl-flip` :eyes: 

It seems `KIcon` has some handling for RTL language flipping. I could see 3 potential circumstances then:
1) This does nothing that `KIcon` doesn't already do and the class should be removed.
1) This particular code might be breaking and need `rtl-flip` removed.
2) Other cases where icons weren't flipped originally, might not always be best if flipped.

Edit: I saw the `Icon` component, and that it has some handling for RTL flipping. It seems `KIcon` has this information on a per-icon basis, so (3) shouldn't be a problem if we're using `KIcon` everywhere.

Reply:
Ahh yes. I checked removing it will not cause any issues. Made the required changes.